gradlew

```diff
@@ -1,17 +1,35 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Are you sure the script has no bash'isms (since `sh` may be anything form old-school Bourne shell to dash)?

Reply:
automatically generated by `./gradlew wrapper --gradle-version=4.4.1 --distribution-type=bin`